loader/loader.go

```diff
@@ -141,7 +141,7 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, workerWg *
 					threshold, _ := val.(int)
 					if job.offset >= int64(threshold) {
 						log.Warnf("[failpoint] load offset %d execeeds threshold %d, will panic", job.offset, threshold)
-						panic("LoadExceedOffsetExit")
+						utils.OsExit(1)
```

Comment:
can `utils.OsExit` carry some msg like `LoadExceedOffsetExit`?

Reply:
The error message is not necessary, as we can log enough information before `utils.OsExit`, besides the function type `OsExit func(int)` is more natural to an `os.Exit` mock